### PR TITLE
- revert to 6.6

### PR DESCRIPTION
**Pull request by labibdotc**

### CODE REVIEW

File: go-oapi-codegen/pom.xml [hunk 1]
```diff
@@ -121,7 +121,7 @@
     </dependencies>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <openapi-generator-version>7.9.0</openapi-generator-version>
+        <openapi-generator-version>6.0.0</openapi-generator-version>
```
Comment (ptc-mjacob) [inline go-oapi-codegen/pom.xml:124]: Should be 6.6.0

File: go-oapi-codegen/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <artifactId>go-oapi-codegen-openapi-generator</artifactId>
     <packaging>jar</packaging>
     <name>go-oapi-codegen-openapi-generator</name>
-    <version>0.1.16</version>
+    <version>0.1.15</version>
```
Comment (ptc-mjacob) [inline go-oapi-codegen/pom.xml:8]: I don't think we should lower the version number.

File: go-oapi-codegen/release/go-oapi-codegen-0.1.16.jar
Comment (ptc-mjacob) [inline go-oapi-codegen/release/go-oapi-codegen-0.1.16.jar:1]: I wouldn't remove it. We can revert the latest binary to 0.1.15. We could also just update it in bindgen config in go-client.

**Review by ptc-mjacob — CHANGES_REQUESTED** [review]

File: go-oapi-codegen/release/go-oapi-codegen-0.1.16.jar
Comment (labibdotc) [reply go-oapi-codegen/release/go-oapi-codegen-0.1.16.jar:1]: The custom generator 0.1.16 is un-deterministically broken.

File: go-oapi-codegen/pom.xml (hunk 1 above)
Comment (labibdotc) [reply go-oapi-codegen/pom.xml:124]: I notice. It is what it used to be and 6.6.0 breaks the `mvn package`

**Pull request closed without merging**